great_expectations/core/expectation_suite.py

```diff
@@ -150,6 +150,18 @@ def expectation_suite_name(self) -> str:
     def expectation_suite_name(self, value) -> None:
         self.name = value
 
+    @property
+    def evaluation_parameter_options(self) -> tuple[str, ...]:
```

Comment:
i'm curious, how does `tuple[str, ...]` differ from `tuple[str]`?

Reply:
`tuple[str]` is a tuple with exactly one element of type string. The syntax I used is a tuple of any length where all elements are of type string.